src/components/Header/Header.tsx

```diff
@@ -1,156 +1,173 @@
-            size="sm"
-            endDecorator={(
-              <Typography
-                component="span"
-                variant="outlined"
-                level="body3"
-                sx={{ bgcolor: 'background.surface', mx: 0 }}
-              >
-                ⌘K
-              </Typography>
-            )}
-            sx={{
-              '--Input-radius': '40px',
-              '--Input-paddingInline': '12px',
-              width: 160,
-              display: { xs: 'none', lg: 'flex' },
-            }}
-          />
-          <Badge badgeContent={2} variant="solid" color="danger">
-            <IconButton variant="soft" sx={{ borderRadius: 'xl' }}>
-              <NotificationsIcon />
-            </IconButton>
+      id={mobileMenuId}
+      keepMounted
+      transformOrigin={{
+        vertical: 'top',
+        horizontal: 'right',
+      }}
+      open={isMobileMenuOpen}
+      onClose={handleMobileMenuClose}
+    >
+      <MenuItem>
+        <IconButtonNew>
+          <Badge badgeContent={4} color="error">
+            <MailIcon />
           </Badge>
-          <IconButton sx={{ borderRadius: 'xl' }}>
-            <Avatar src="/static/images/avatar/1.jpg" size="sm" />
-            <KeyboardArrowDownIcon />
-          </IconButton>
-        </Box>
-      </Box>
-    </Sheet>
+        </IconButtonNew>
+        <p>Messages</p>
+      </MenuItem>
+      <MenuItem>
+        <IconButtonNewOne>
+          <Badge badgeContent={17} color="error">
+            <NotificationsIcon />
+          </Badge>
+        </IconButtonNewOne>
+        <p>Notifications</p>
+      </MenuItem>
+      <MenuItem onClick={handleProfileMenuOpen}>
+        <IconButtonNewTwo>
+          <AccountCircle />
+        </IconButtonNewTwo>
+        <p>Profile</p>
+      </MenuItem>
+    </Menu>
+  );
+
+  return (
+    <BoxNew>
+      <AppBarNew>
+        <Toolbar>
+          <FormGroup>
+            <FormControlLabel
+              control={(
+                <SwitchNew
+                  checked={auth}
+                  onChange={handleChange}
+                />
+              )}
+              label={auth ? 'Logout' : 'Login'}
+            />
+          </FormGroup>
+          <TypographyNew>KIT</TypographyNew>
+          <Search>
+            <SearchIconWrapper>
+              <SearchIcon />
+            </SearchIconWrapper>
+            <StyledInputBaseNew />
+          </Search>
+          <BoxNew />
+          <BoxNewOne>
+            <IconButtonFour>
+              <Badge badgeContent={4} color="error">
+                <MailIcon />
+              </Badge>
+            </IconButtonFour>
+            <IconButtonFive>
+              <Badge badgeContent={17} color="error">
+                <NotificationsIcon />
+              </Badge>
+            </IconButtonFive>
+            <IconButtonSix onClick={handleProfileMenuOpen}>
+              <AccountCircle />
+            </IconButtonSix>
+          </BoxNewOne>
+          <BoxNewTwo>
```

Comment:
`BoxNewTwo` it does not say anything. 

It is also related to many other weir names

Reply:
add correction